project/build.properties

```diff
@@ -15,4 +15,4 @@
 # limitations under the License.
 #
 # Please update the version in appveyor-install-dependencies.ps1 together.
-sbt.version=1.7.2
```

Comment:
For the convenience of others manual check, it will be revert later

Reply:
[fdc2067](https://github.com/apache/spark/pull/38481/commits/fdc2067873d199f1fa29011d7b4be8bff415ca13) revert sbt version to 1.7.2, will another pr for re upgrade. Please manually modify `project/build.properties` when you double check @linhongliu-db